katdal/test/test_lazy_indexer.py

```diff
@@ -73,19 +74,182 @@ def test_too_many_axes(self):
         self._test_index_error(np.s_[0, 0, 0, 0])
 
 
+def slice_to_range(s, l):
+    return range(*s.indices(l))
```

Comment:
Should add `from builtins import range` so that you get the same behaviour in Python 2 and 3.

Reply:
Done.